ironfish-cli/src/commands/reset.ts

```diff
@@ -45,6 +45,9 @@ export default class Reset extends IronfishCommand {
       this.exit(0)
     }
 
```

Comment:
We should get default from `this.sdk.config.defaults.networkId` right?

Reply:
Yep, that looks right!